src/Auth.js

```diff
@@ -21,11 +21,11 @@ function Auth({ config, isMaster = false, isReadOnly = false, user, installation
 
 // Whether this auth could possibly modify the given user id.
 // It still could be forbidden via ACLs even if this returns true.
-Auth.prototype.couldUpdateUserId = function(userId) {
+Auth.prototype.couldUpdateUserId = function() {
```

Comment:
so this is confusing to me now.  the name `couldUpdateUserId` with a userId argument the way it was made sense why this would exist, but I can no longer really understand.  Is this just loggedInOrMaster?

Reply:
you are right :)